app/components/Utility/HelpContent.jsx

```diff
@@ -140,6 +140,7 @@ class HelpContent extends React.PureComponent {
                 `missing path '${this.props.path}' for locale '${locale}' help files, rolling back to alt_path '${this.props.alt_path}'`
             );
             value = HelpData[locale][this.props.alt_path];
+            if (!value) value = HelpData["en"][this.props.alt_path];
```

Comment:
According to the old logic, it seems the order to try need to be
1. locale.path
2. locale.alt_path
3. en.path
4. en.alt_path

This patch changed the order.

And it seems that the old code was correct.

The code `locale != "en"` in line 153 looks suspicious. Should it be `locale !== "en"` (to be consistent with line 146)?

The code `return !null` in line 164 looks suspicious too. Should it be `return null`?

Reply:
It looks like this code (line 167-178) failed to extract the desired section and re-assign it to `value`
```
                    value = key.replace(
                        new RegExp("^" + this.props.section + ","),
                        ""
                    );
```
thus in the end the `setVars` call tries to set a bad `value` and caused the recursion?
```
                dangerouslySetInnerHTML={{
                    __html: this.setVars(value, this.props.hide_issuer)
                }}
```

Reply:
I've verified that even commenting out dangerouslySetInnerHTML, the same will loop infinitely.